resources/tutorials/running_production.md

```diff
@@ -26,10 +26,10 @@ Using MeiliSearch on your own machine for your weekend project is fun, let's agr
 + [3.2. Set up SSL/TLS for your MeiliSearch](/resources/tutorials/running_production.md#_3-2-set-up-ssl-tls-for-your-meilisearch)
 
 [Conclusion](/resources/tutorials/running_production.md#conclusion)
-  
+
 ## Get your MeiliSearch ready for production
-  
-For this tutorial, we will be using a Debian 10 server, running on DigitalOcean. You can easily try it on your own, with plans starting at $5/month. And if you want some credits to start running your MeiliSearch and are not already registered on DigitalOcean, you can get $100 for free using [this referral link](https://m.do.co/c/7c67bd97e101).  
+
+For this tutorial, we will be using a Debian 10 server, running on DigitalOcean. You can easily try it on your own, with plans starting at $5/month. And if you want some credits to start running your MeiliSearch and are not already registered on DigitalOcean, you can get $100 for free using [this referral link](https://m.do.co/c/7c67bd97e101).
```

Comment:
ça parraine sec!

Reply:
That's not my tutorial but only whitespaces removed from the @eskombro article 😉